gallery/src/guidelines/ColorPalette/ColorPalettePage.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import Swatcher from './ColorPaletteExample';
+
+const ColorPalettePage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        The design group have created a palette of colors for use in RSC. The colors are organized into swatches
+        based on hue (red, green, blue, etc) and assigned values based on their lightness or darkness. Darker colors
+        have higher values, and lighter colors have lower values, for example the lightest red is
+        {' '}<code className="nx-code">$nx-red-50</code> and the darkest is
+        {' '}<code className="nx-code">$nx-red-1000</code>. All of the color swatches are located in
+        {' '}<code className="nx-code">_nx-color-swatches.scss</code>. The swatches below show all of the colors
+        available in RSC along with their hex value and corresponding RSC variable.
+      </p>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Color Variables</h3>
+        </header>
+        <p className="nx-p">
+          Where a color is used commonly across multiple components a custom variable is created. For example many
+          components with a disabled state have the same background color so we have created a color variable:
+          {' '}<code className="nx-code">$nx-disabled-background: $nx-grey-200;</code>. If the disabled background color
```

Comment:
I'm not sure we should mention this unless we are going to document what additional variables are supported.  Especially since `_nx-colors.scss` currently contains a mix of up-to-date semantic color variables, and deprecated color variables that aren't part of the current system and which we are theoretically working on removing.

Reply:
> I'm not sure we should mention this unless we are going to document what additional variables are supported. Especially since `_nx-colors.scss` currently contains a mix of up-to-date semantic color variables, and deprecated color variables that aren't part of the current system and which we are theoretically working on removing.

My goal was to explain why sometimes we use `$nx-disabled-background` instead of `$nx-grey-200` (for example), I agree that we don't want to list all of the named variables, but I do think that the explanation has value.

Reply:
> My goal was to explain why sometimes we use $nx-disabled-background instead of $nx-grey-200 (for example), I agree that we don't want to list all of the named variables, but I do think that the explanation has value.

I'm worried that it prompts users to use colors from `_nx-colors.scss` and that's not currently a safe idea.

Reply:
> > My goal was to explain why sometimes we use $nx-disabled-background instead of $nx-grey-200 (for example), I agree that we don't want to list all of the named variables, but I do think that the explanation has value.
> 
> I'm worried that it prompts users to use colors from `_nx-colors.scss` and that's not currently a safe idea.

The problem is that there are many variables in `_color.scss` that we _do_ want people to use rather than a `nx-color-###` variable. I think that this isn't a problem with this documentation but rather a problem with `_colors.scss`. I know that we had a discussion a little while ago about moving some of those variables down to the bottom of the file into a "deprecated" section. Maybe we should up the priority of that task.

Reply:
> The problem is that there are many variables in _color.scss that we do want people to use rather than a nx-color-### variable. I think that this isn't a problem with this documentation but rather a problem with _colors.scss. I know that we had a discussion a little while ago about moving some of those variables down to the bottom of the file into a "deprecated" section. Maybe we should up the priority of that task.

Right.  But until we clean that up, I think we shouldn't document it in the way you have here.  We can make sure we document the desirable variables later once they are distinguishable.

Reply:
> > The problem is that there are many variables in _color.scss that we do want people to use rather than a nx-color-### variable. I think that this isn't a problem with this documentation but rather a problem with _colors.scss. I know that we had a discussion a little while ago about moving some of those variables down to the bottom of the file into a "deprecated" section. Maybe we should up the priority of that task.
> 
> Right. But until we clean that up, I think we shouldn't document it in the way you have here. We can make sure we document the desirable variables later once they are distinguishable.

Removed here and I created: https://issues.sonatype.org/browse/RSC-496